src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.0.props

```diff
@@ -13,10 +13,13 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>1.0.1</Version>
+      <Version>1.0.5</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>1.0.4</Version>
+      <Version>1.0.7</Version>
+    </NETCoreApp10Dependency>
+    <NETCoreApp10Dependency Include="Microsoft.NETCore.Targets">
```

Comment:
I don't see this dependency on [https://www.nuget.org/packages/Microsoft.NETCore.App/1.0.5](https://www.nuget.org/packages/Microsoft.NETCore.App/1.0.5)
But I DO see a dependency for Microsoft.NETCore.Platforms (>= 1.0.2), which I don't see here.